packages/core/src/user-store/sqlite/store.ts

```diff
@@ -149,6 +149,23 @@ export class SqliteUserStore implements UserStore {
     });
   };
 
+  linkToCurrent = async () => {
+    if (!this.schema) return;
+
+    await this.db.transaction().execute(async (tx) => {
+      await Promise.all(
+        this.schema!.entities.map(async (model) => {
+          const tableName = `${model.name}_${this.versionId}`;
+          tx.schema
+            .createView(model.name)
+            .orReplace()
+            .as(tx.selectFrom(tableName).selectAll())
+            .execute();
+        })
+      );
+    });
```

Comment:
It seems like these views could be created just once during `reload`. Or am I missing something?